JsonSchema/SchemaRegistry.cs

```diff
@@ -30,15 +28,25 @@ private class Registration
 		}
 
 		private static readonly Uri _empty = new Uri("http://everything.json/");
+		private static readonly SchemaRegistry _global;
 
 		private Dictionary<Uri, Registration>? _registered;
 		private Func<Uri, JsonSchema?>? _fetch;
 		private Stack<Uri>? _scopes;
+		private ValidationOptions? _options;
+
 
 		/// <summary>
 		/// The global registry.
 		/// </summary>
-		public static SchemaRegistry Global { get; }
+		public static SchemaRegistry Global
+		{
+			get
+			{
+				_global._options ??= ValidationOptions.Default;
+				return _global;
+			}
+		}
```

Comment:
Rather than initialize Global right away with the options, we wait until it's accessed.  By this time, the static `ValidationOptions.Default` will have been initialized, and we can safely set it here.